im-gespraech-buber-gesellschaft.csl

```diff
@@ -0,0 +1,651 @@
+    <layout prefix="(" suffix=")" delimiter="; ">
+      <choose>
+        <if match="any" position="ibid-with-locator">
+          <text term="ibid" suffix=" "/>
+          <text macro="citation-locator"/>
+        </if>
+        <else-if match="any" position="ibid">
+          <text term="ibid"/>
+        </else-if>
+        <else>
+          <group>
+            <choose>
+              <if match="any" variable="title-short">
+                <text variable="title-short"/>
+              </if>
+              <else>
+                <group delimiter=" ">
+                  <text macro="author-short"/>
+                  <choose>
+                    <if type="webpage" match="all">
+                      <choose>
+                        <if match="any" variable="issued">
+                          <date form="text" date-parts="year-month-day" variable="issued">
+                            <date-part name="year"/>
+                          </date>
+                        </if>
+                        <else>
+                          <choose>
+                            <if match="any" variable="accessed">
+                              <group>
+                                <text term="accessed"/>
+                                <date form="numeric" variable="accessed" prefix=" ">
+                                  <date-part name="day" form="numeric-leading-zeros"/>
+                                  <date-part name="month" form="numeric-leading-zeros"/>
+                                  <date-part name="year"/>
+                                </date>
+                              </group>
+                            </if>
+                            <else>
+                              <text term="no date"/>
+                            </else>
+                          </choose>
+                        </else>
+                      </choose>
+                    </if>
+                    <else>
+                      <text macro="year-date"/>
+                    </else>
+                  </choose>
+                  <choose>
+                    <if type="manuscript" match="all" variable="archive">
+                      <text macro="archive"/>
+                    </if>
+                  </choose>
+                </group>
+              </else>
+            </choose>
+            <group>
+              <text macro="citation-locator" prefix=", "/>
+            </group>
+          </group>
+        </else>
+      </choose>
+    </layout>
+  </citation>
+  <bibliography et-al-min="9" et-al-use-first="8" subsequent-author-substitute="-" subsequent-author-substitute-rule="complete-each" hanging-indent="true">
+    <sort>
+      <key macro="author"/>
+      <key macro="year-date"/>
+      <key variable="title"/>
+    </sort>
+    <layout suffix=".">
+      <choose>
+        <if position="ibid-with-locator">
```

Comment:
I'm surprised validation doesn't fail here, but you can't test for position in the bibliography.